examples/arithmetic2/README.md

```diff
@@ -0,0 +1,2 @@
+same as problem arithmetic, but with some source files decorated with 
+`@EXPECTED_GRADES@` to serve as a test suite for analyzetestgroups.
```

Comment:
change the `arithmetic` problem instead (it's ok to modify it freely)

Reply:
That's what I did first. Problem is that it contains a deliberately false @EXPECTED_VALUE@, in order to trigger  the script's warning.
```
WARNING: arithmetic_ld.cpp: Unexpected grade AC on test group 3. (Expected WA).
```
I found that confusing. But I can add some more comments.

Reply:
Actually, I should just add another submission that makes these kinds of mistakes and leave `arithmetic_ld.cpp` unmolested.

Reply:
Done in bfe1ea9